### ISSUE TITLE
Softmax function not being applied? : PyTorch Code

### ISSUE THREAD

**Issue by rithinch** [CRIPAC-DIG/SR-GNN#16]
In the paper it is mentioned that the **softmax function** is being applied to the outputs after computing scores for each candidate item (_**equation 9**_), however, the PyTroch Implemention ([compute_scores](https://github.com/CRIPAC-DIG/SR-GNN/blob/9eff4b7fde42d891b5aa379ff4dbe0af48b2c68c/pytorch_code/model.py#L89) method) doesn't seem to be doing that. Is there a reason why it isn't used?

I have tested the model with the softmax function (default hyperparameters), and the evaulation metrics peformance has drastically decreased. I suspect if it has anything to do with the low output values after softmax. Any thoughts on this?

**Comment by RuihongQiu** [comment]
[nn.CrossEntropyLoss()](https://pytorch.org/docs/stable/nn.html#crossentropyloss)

**Comment by rithinch** [comment]
Thanks for clarifying. @RuihongQiu
